internal/launcher/analyze.go

```diff
@@ -52,35 +56,46 @@ func analyze(gs *state.GlobalState, args []string) (k6deps.Dependencies, error)
 	return k6deps.Analyze(dopts)
 }
 
-// scriptNameFromArgs returns the file name passed as input and true if it's a valid script name
-func scriptNameFromArgs(args []string) (string, bool) {
+// search for the command and returns a boolean indicating it takes a script or not
+func requiresScript(args []string) bool {
 	// return early if no arguments passed
 	if len(args) == 0 {
-		return "", false
+		return false
 	}
```

Comment:
It confuses me, what does it happen if it is required, but the user didn't pass any for a mistake?

Reply:
Yes. I don't think we should make this validation here.